license-maven-plugin/src/main/java/com/mycila/maven/plugin/license/AbstractLicenseMojo.java

```diff
@@ -295,6 +296,15 @@
   @Parameter(property = "license.skip", defaultValue = "false")
   public boolean skip = false;
 
+  /**
+   * Whether to skip the plugin execution on a shallow git clone.
+   * <p>
+   * We recommend setting this to {@code true} if you are generating copyright
+   * ranges from file creation dates in git history.
+   */
+  @Parameter(property = "license.skipIfShallow", defaultValue = "false")
+  public boolean skipIfShallow = false;
```

Comment:
@dbwiddis Remove the = false here as that is java default and when defined is initializing twice.

Reply:
OK but I copied verbatim from `skip` 8 lines above, and `quiet` 8 lines below is the same format. Should I fix those too?

Reply:
I'm going to be fixing all those issues shortly :)